insonmnia/benchmarks/benchmarks.go

```diff
@@ -87,15 +89,15 @@ func (bl *benchmarkList) readResults(ctx context.Context, reader io.ReadCloser)
 		return fmt.Errorf("cannot decode JSON response: %v", err)
 	}
 
-	var max uint64
+	bl.byID = make([]*pb.Benchmark, len(data))
```

Comment:
This logic is wrong when we remove some benchmarks or enumerate them using random offset.

Reply:
Emm, what?

For example you have a json of:
```json
{
  "b1": {"id": 0},
  "b2": {"id": 2},
}
```

It's not malformed. Why it is? Otherwise we should reenumarate all benchmarks just because one day we removed "id": 1, in the middle, because of reasons.
Moreover, all orders that are currently in the market should be invalidated.
This is shit.